operations/locustfile.py

```diff
@@ -74,14 +76,22 @@ def post_v1_etor_orders(self):
         if response.status_code == 200:
             self.orders_api_called = True
 
-    @task(2)
+    @task(1)
     def get_v1_etor_metadata(self):
         if self.orders_api_called:
             self.client.get(
                 f"{METADATA_ENDPOINT}/{self.submission_id}",
                 headers={"Authorization": self.access_token},
             )
 
```

Comment:
I assume this will be added later on for `CONSOLIDATED_ENDPOINT`? Or was this accidentally commented out?

Reply:
Resolving this as I commented in the wrong spont.